Rakefile

```diff
@@ -4,10 +4,12 @@ require "bundler/gem_tasks"
 # RSpec
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new
-task :default => ['dummy:create', :spec]
+task :default => ['testapp:create', :spec]
```

Comment:
Good! :+1: 

How about a consistent name like `testapp`? We could move it into a folder `testapp` and the database would be named `chronomodel_testapp`?

Reply:
Agreed.